esp/services/ws_workunits/ws_workunitsQuerySets.cpp

```diff
@@ -766,11 +766,21 @@ class QueryFileCopier
         if (queryname && *queryname)
             queryname = queryid.append(queryname).append(".0").str(); //prepublish dummy version number to support fuzzy match like queries="myquery.*" in package
         files->addFilesFromQuery(cw, pm, queryname);
+        Owned<IDFUhelper> helper = createIDFUhelper();
+#ifdef _CONTAINERIZED
+        //Find the first data plane
```

Comment:
shouldn't the target plane be configurable?
Isn't the 1st plane likely not to be the right one?

Reply:
yes it should.  I will open that as a separate jira.  This is struggling to get it to work at all.

Reply:
@ghalliday - was a JIRA opened for this?

Reply:
It was implemented by the last commit - it now takes it from the roxie storagePlane if defined.

Reply:
ok thanks